docs/static/ls-to-serverless.asciidoc

```diff
@@ -6,13 +6,13 @@ When you use Elasticsearch on Elastic Cloud Serverless you don’t need to worry
 
 .{ls} to {serverless-full}
 ****
-You’ll use the {ls} link:logstash-docs-md://lsr/plugins-outputs-elasticsearch.md[{es} output plugin] to send data to {serverless-full}.
+You'll use the {ls} link:https://www.elastic.co/guide/en/logstash/8.19/plugins-outputs-elasticsearch.html[{es} output plugin] to send data to {serverless-full}.
 Note these differences between {es-serverless} and both {ech} and self-managed {es}:
 
-* Use link:/reference/secure-connection.md#ls-api-keys[**API keys**] to access {serverless-full} from {ls} as it does not support native user authentication.
-  Any user-based security settings in your link:logstash-docs-md://lsr/plugins-outputs-elasticsearch.md[{es} output plugin] configuration are ignored and may cause errors.
-* {serverless-full} uses **data streams** and link:docs-content://manage-data/lifecycle/data-stream.md[{dlm} ({dlm-init})] instead of {ilm} ({ilm-init}). Any {ilm-init} settings in your link:logstash-docs-md://lsr/plugins-outputs-elasticsearch.md[{es} output plugin] configuration are ignored and may cause errors.
-* **{ls} monitoring** is available through the link:https://github.com/elastic/integrations/blob/main/packages/logstash/_dev/build/docs/README.md[{ls} Integration] in link:docs-content://solutions/observability.md[Elastic Observability] on {serverless-full}.
+* Use link:https://www.elastic.co/guide/en/logstash/8.19/ls-security.html#ls-api-keys[API keys] to access {serverless-full} from {ls} as it does not support native user authentication.
+  Any user-based security settings in your {ls} link:https://www.elastic.co/guide/en/logstash/8.19/plugins-outputs-elasticsearch.html[{es} output plugin] configuration are ignored and may cause errors.
+* {serverless-full} uses **data streams** and link:https://www.elastic.co/guide/en/elasticsearch/reference/8.19/data-stream-lifecycle.html[{dlm} ({dlm-init})] instead of {ilm} ({ilm-init}). Any {ilm-init} settings in your link:https://www.elastic.co/guide/en/logstash/8.19/plugins-outputs-elasticsearch.html[{es} output plugin] configuration are ignored and may cause errors.
```

Comment:
```suggestion
* {serverless-full} uses **data streams** and link:https://www.elastic.co/guide/en/elasticsearch/reference/8.18/data-stream-lifecycle.html[{dlm} ({dlm-init})] instead of {ilm} ({ilm-init}). Any {ilm-init} settings in your link:https://www.elastic.co/guide/en/logstash/8.19/plugins-outputs-elasticsearch.html[{es} output plugin] configuration are ignored and may cause errors.
```